src/helpers/buildFilename.spec.js

```diff
@@ -0,0 +1,59 @@
+import { buildFilename } from 'src/helpers/buildFilename'
+
+describe('buildFilename', () => {
+  it('should replace "/" by "|" in qualificationName', () => {
+    const paperName = buildFilename({
+      qualificationName: 'passport/test'
+    })
+
+    expect(paperName).toEqual('passport|test.pdf')
```

Comment:
@taratatach Do we have a list of characters that may cause troubles for Desktop somewhere?

Reply:
Yes we do.

On Windows :
- forbidden chars are `<>:"/\|?*`
- forbidden last chars are `. `
- forbidden names are 'CON', 'PRN', 'AUX', 'NUL', 'COM1', 'COM2', 'COM3', 'COM4', 'COM5', 'COM6', 'COM7', 'COM8', 'COM9', 'LPT1', 'LPT2', 'LPT3', 'LPT4', 'LPT5', 'LPT6', 'LPT7', 'LPT8', 'LPT9'

On macOS and Linux : 
- forbidden chars are `/`

These can be found here: https://github.com/cozy-labs/cozy-desktop/blob/master/core/incompatibilities/platform.js#L110-L160

Reply:
I knew it! Thank, I'll bookmark this file :D

Reply:
Ok, I'll replace that.
Thx for the informations 👍